packages/teleterm/package.json

```diff
@@ -55,7 +55,7 @@
   },
   "productName": "Teleport Connect",
   "build": {
-    "appId": "gravitational.teleport.connect",
+    "appId": "com.gravitational.teleport.connect",
```

Comment:
@gzdunek Most of the changes probably will make more sense to Alan, but off the top of your head do you remember if changing this value is going to have any impact on the app?